testsuite/MDAnalysisTests/analysis/test_hbonds.py

```diff
@@ -59,6 +59,16 @@ def setUp(self):
             'acceptor_resnm': np.array(['ALA', 'ALA', 'ALA', 'ALA', 'ALA', 'PRO', 'ALA', 'ALA'], dtype='U4'),
             }
 
+    #issue 1229        
+    def test_get_bonded_hydrogen_list(self, **kwargs):
+    	kw = self.kwargs.copy()
+    	kw.update(kwargs)
+    	h = MDAnalysis.analysis.hbonds.HydrogenBondAnalysis(self.universe, **kw)
+    	self.universe.atoms[0].index = -2
```

Comment:
I would like it if the test could be closer to the actual case that trigger the exception. A negative atom index is not realistic. Instead, an index to close to the end of the protein is what the exception seems to be about based on the comment next to it.